package.json

```diff
@@ -101,7 +107,8 @@
   "packageManager": "pnpm@9.15.3",
   "resolutions": {
     "@nuxtjs/tailwindcss": "link:.",
-    "@nuxt/ui": "npm:@nuxt/ui-edge"
+    "@nuxt/ui": "npm:@nuxt/ui-edge",
+    "unicorn-magic": "0.2.0"
```

Comment:
Hi @ineshbose, do you happen to have more info on why this resolution is necessary? I noticed after updating to 6.14.0 that the docker builds for our app broke, but adding this resolution to our package.json fixed things. But that was just a random troubleshooting step, and I'd be curious about the root cause.

Reply:
Hey - yes it is an odd issue. You can do down this thread to investigate https://github.com/nuxt-modules/tailwindcss/issues/954, but even I would be slightly confused 😅